packages/protocol/contracts/governance/GovernanceSlasher.sol

```diff
@@ -70,6 +108,55 @@ contract GovernanceSlasher is Ownable, Initializable, UsingRegistry {
     return true;
   }
 
+  /**
+   * @notice Calls `LockedGold.slash` on `account` if `account` has an entry in `slashed`.
+   * @param account Account to slash
+   * @param electionLessers Lesser pointers for slashing locked election gold.
+   * @param electionGreaters Greater pointers for slashing locked election gold.
+   * @param electionIndices Indices of groups voted by slashed account.
+   */
+  function slashL2(
+    address account,
+    address group,
+    address[] calldata electionLessers,
+    address[] calldata electionGreaters,
+    uint256[] calldata electionIndices
+  ) external onlyL2 onlyAuthorizedToSlash returns (bool) {
+    uint256 penalty = slashed[account];
+    require(penalty > 0, "No penalty given by governance");
+    slashed[account] = 0;
+
+    ILockedGold lockedGold = getLockedGold();
+
+    lockedGold.slash(
+      account,
+      penalty,
+      address(0),
+      0,
+      electionLessers,
+      electionGreaters,
+      electionIndices
+    );
+
+    if (group != address(0)) {
+      lockedGold.slash(
+        group,
+        penalty,
+        address(0),
```

Comment:
Should the reporter be Governance ? or possibly slasherExecuter ?

Reply:
the reporter is meant to get a reward for in the case of downtime slasher for taking the effort of reporting it, here is a multisig that it's permissioned to do this, so it doesn't make sense IMO to incentivate it to slash people